src/server.coffee

```diff
@@ -14,6 +17,7 @@ config.alerts = config.get('alerts')
 config.polling = config.get('polling')
 config.reports = config.get('reports')
 config.auditing = config.get('auditing')
+config.agenda = config.get('agenda')
 
 mongoose = require "mongoose"
 exports.connectionDefault = connectionDefault = mongoose.createConnection(config.mongo.url)
```

Comment:
Aah okay, yeah if it's a mission than definitely skip it. I'm sure it's not too big a deal ito resource usage.